src/Tools/BuildBoss/ProjectUtil.cs

```diff
@@ -136,13 +136,28 @@ internal IEnumerable<XElement> GetItemGroup()
             return _document.XPathSelectElements("//mb:ItemGroup", _manager);
         }
 
-        internal List<ProjectKey> GetDeclaredProjectReferences()
+        internal List<ProjectKey> GetDeclaredProjectReferences(bool withoutRefOutputFalse = true)
```

Comment:
Name is a bit confusing.  Consider `includeReferencOutputFalse` as the flag name.